drizzlepac/haputils/align_utils.py

```diff
@@ -725,6 +730,87 @@ def find_alignment_sources(self, output=True, dqname='DQ', crclean=False,
             self.imghdu.close()
             self.imghdu = None
 
+class SBCHAPImage(HAPImage):
+    def __init__(self, filename):
+        super().__init__(filename)
+        self.exptime = max(self.imghdu[0].header['exptime'], 1.0)
+
+    def find_alignment_sources(self, output=True, dqname='DQ', crclean=False,
+                               **alignment_pars):
+        """Find sources in all chips for this exposure."""
+        # Only 1 chip, no need to loop
+        chip = 1
+
+        # find sources in image
+        if output:
+            outroot = '{}_sci{}_src'.format(self.rootname, chip)
+        else:
+            outroot = None
+
+        sciarr = self.imghdu[("SCI", chip)].data.copy()
+
+        # Remove all background noise
+        # This background noise is effectively integerized by the SBC detector
+        sci_gauss = ndimage.gaussian_filter(sciarr, sigma=3.0)
+        bkg_noise = astropy.stats.sigma_clipped_stats(sci_gauss, maxiters=1)
+        # bkg_noise = astropy.stats.sigma_clipped_stats(sciarr[sciarr > 0], maxiters=1)
+        bkg_limit = bkg_noise[1] + 3 * bkg_noise[2]  # Set limit as 1-sigma above mean
+        log.debug(f"BKG_LIMIT for source identification: {bkg_limit} based on {bkg_noise}")
+        # create mask of all background pixels and zero out all those pixels
+        bkg_mask = sci_gauss <= bkg_limit
+        sci_bkgsub = sciarr.copy()
+        sci_bkgsub[bkg_mask] = 0
+
+        # A high LoG sigma value is needed to smooth over the pixelated nature of the SBC PSFs
+        # This avoids having PSF halo peaks being detected as 'real' sources
+        slabels, snum, logimg = amutils.detect_point_sources(sci_bkgsub, background=0, log_sigma=5.0)
+        # create mask based on these sources only...
```

Comment:
Line 766. If "logimg" is really not ever used, replace with "_".

Reply:
Fixed in 7b869e6.